services/web/server/tests/integration/01/utils.py

```diff
@@ -0,0 +1,67 @@
+import sys
+from pathlib import Path
+from typing import List, Tuple
+from uuid import UUID
+
+from models_library.projects import ProjectID
+from pytest_simcore.helpers.utils_login import AUserDict, log_client_in
+from simcore_service_webserver.security_roles import UserRole
+from yarl import URL
+
+CURRENT_DIR = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
+
+DATA_DIR = CURRENT_DIR.parent.parent / "data"
```

Comment:
Having to do ``DATA_DIR = CURRENT_DIR.parent.parent / "data"`` is a good example of why it is more maintainable to have fixtures ... 

As stated above, I would recommend having fixtures


```python
@pytest.fixture
def login_user_and_import_study(...) -> Callable:
     async def doit(...):
          ...
     return doit

# alternatively you can create another intermediary fixture here and pass the result to the test below

def test_it(login_user_and_import_study: Callable):
     await login_user_and_import_study(...)
```

Reply:
changed as discussed during pair review.